### ISSUE TITLE
How is DynSynLayer nn.Linear params Updated

### ISSUE THREAD

**Issue by InckieUltra** [Beanpow/DynSyn#1]
Hi, Kaibo He.

DynSyn is very insightful !

After reading your code, I have a question. 

There is a nn.Linear layer in DynSyn layer, I wonder how is the params of this layer updated? 
I thought it is a part of policy. However, the policy loss is calculated by only raw action and raw log_prob.

I'm new to reinforcement learning, so I wonder how is the params of this layer updated?

Thanks!
